src/emulate.c

```diff
@@ -9,6 +9,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include "common.h"
```

Comment:
There is no need to include `common.h` since the build system already specifies it as the first header for inclusion.